src/main.js

```diff
@@ -209,10 +214,11 @@ export async function convertToQuorum(argv) {
   console.log('Conversion to Quorum Vat complete.');
 }
 
-export async function buildArgv(vat, argvJSON, readBaseFile, vatEndowments) {
+export function buildArgv(vat, argvJSON, readBaseFile, vatEndowments) {
```

Comment:
Hrm, I don't think we can remove the 'async' from this function and still have it behave the same way. The `readBaseFile` uses await because filesystem access is async, and the promise returned by `buildArgv` should not fire until all the argv values are available.

Reply:
Ah good point! I only learned this recently about forEach.